packages/api/src/services/actions.ts

```diff
@@ -71,51 +72,58 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       assert(exists(emp.tokenCurrency), "EMP does not have token currency address");
       return queries.historicalPricesByTokenAddress(emp.tokenCurrency, start, end);
     },
+    async historicalCollateralPrices(empAddress: string, start = 0, end: number = Date.now()): Promise<PriceSample[]> {
+      assert(empAddress, "requires an empAddress");
+      const emp = await queries.getAnyEmp(empAddress);
+      assert(exists(emp.collateralCurrency), "EMP does not have token currency address");
+      return queries.historicalPricesByTokenAddress(emp.collateralCurrency, start, end);
+    },
```

Comment:
this function was moved and not changed, just so i could line it up with the readme.